packages/manager/modules/pci/src/projects/project/kubernetes/details/audit-logs/audit-logs.html

```diff
@@ -9,74 +9,6 @@
         <span class="oui-icon oui-icon-external-link" aria-hidden="true"></span>
     </a>
 
-    <tail-logs>
-        <div
-            data-ng-repeat="log in $ctrl.logger.logs track by $index"
-            data-ng-class="{
-            'text-warning' : $ctrl.constructor.isWarningResponse(log.message.audit_responseStatus),
-            'text-danger'  : $ctrl.constructor.isDangerResponse(log.message.audit_responseStatus)
-        }"
-        >
-            <div class="row ml-0">
-                <span
-                    class="col-md-1 text-break"
-                    data-ng-bind="::log.message.audit_requestReceivedTimestamp_date"
-                    title="date"
-                ></span>
-                <span
-                    class="col-md-1 text-break"
-                    data-ng-bind="::log.message.audit_verb | uppercase"
-                    title="verb"
-                ></span>
-                <span
-                    class="col-md-1 text-break"
-                    data-ng-bind="::log.message.audit_authorizationDecision | uppercase"
-                    title="authorization_decision"
-                ></span>
-                <span
-                    class="col-md-1 text-break"
-                    data-ng-bind="::log.message.audit_responseStatus"
-                    title="response_status"
-                ></span>
-                <span
-                    class="col-md-1 text-break"
-                    data-ng-bind="::log.message.audit_user"
-                    title="user"
-                ></span>
-                <span
-                    class="col-md-2 text-break"
-                    data-ng-bind="::log.message.audit_requestURI"
-                    title="request_uri"
-                ></span>
-                <span class="col-md-4 text-break">
-                    <span
-                        data-ng-bind="::log.message.audit_groups"
-                        title="groups"
-                    ></span>
-                    <span title="ident"> - </span>
-                    <span
-                        data-ng-bind="::log.message.audit_authorizationReason"
-                        title="authorization_reason"
-                    ></span>
-                    <span title="ident"> - </span>
-                    <span
-                        data-ng-bind="::log.message.audit_userAgent"
-                        title="user_agent"
-                    ></span>
-                </span>
-                <span
-                    class="col-md-1 text-break"
-                    data-ng-bind="::log.message.audit_auditID"
-                    title="audit_id"
-                ></span>
-            </div>
-
-            <br />
-        </div>
-
-        <div
-            data-translate="kube_logs_list_no_logs"
-            data-ng-if="$ctrl.logger.logs.length === 0"
-        ></div>
-    </tail-logs>
+    <log-to-customer source="$ctrl.logSrcUrl" log-keys="$ctrl.LOG_KEYS">
+    </log-to-customer>
```

Comment:
no right tile for now ?

Reply:
No, but it is planned for the next task. Still, I prepared the code with ng-transclude and according CSS already